src/tests/class-task-list.test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import TaskList from '../modules/class-task-list.js';
+
+// Testing Add Item Todo
+
+describe('Testing Add Item', () => {
+  test('Adding todo to array not to be null', () => {
+    const todo = new TaskList();
+    todo.addTask('WakeUp', false, 0);
+
+    expect(todo.length).not.toBeNull();
+  });
+  it('Adding new task', () => {
+    const todo = new TaskList();
+    todo.addTask('Practice', false, 1);
+
```

Comment:
- [ ] Let's add tests for following functions:
- The function for editing the task description.
- The "Clear all completed" function.
- The function for updating an item's 'completed' status.

Reply:
This change has been done well